### PR TITLE
Add more type-annotations

### PR DESCRIPTION
**Pull request by marius92mc**
This PR adds more type-annotations.

### CODE REVIEW

File: tests/test_github.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # pylint: disable=no-member,invalid-name,redefined-outer-name
 import pytest
 import responses
+import typing
```
Comment (marius92mc) [inline tests/test_github.py]: Will fix this mypy complaint.

Comment (ignacio-chiazzo) [reply tests/test_github.py]: that lint is weird, why does `typing` fail and not the other imports?

**Comment by marius92mc** [comment]
@ignacio-chiazzo, I am answering in a new comment because apparently it doesn't let me to comment on that discussion thread. 

I think that it was because it wanted the `typing` package to be before the _installed_ packages section, which comes after the first section of `import`s which is for built-in packages, like `os`, `typing`, `re`, etc. 

~And within each group it expects them to be stored alphabetically.~
Actually looks like it doesn't complain about not putting them in the alphabetical order, for example with [this](https://github.com/marius92mc/github-stargazers/blob/da6924a25282b3bc21cf852c02f8b41f76f275b9/github_stargazers/github.py#L1) case.

**Comment by marius92mc** [comment]
I fixed CI complains and it is fine now.

**Pull request merged**